cli/bpmetadata/types_ui.go

```diff
@@ -58,10 +58,10 @@ type DisplayVariable struct {
 	MaximumLength int `json:"maxLength,omitempty" yaml:"maxLength,omitempty"`
 
 	// Minimum value for numeric types.
-	Minimum int `json:"min,omitempty" yaml:"min,omitempty"`
+	Minimum float32 `json:"min,omitempty" yaml:"min,omitempty"`
```

Comment:
I'm assuming the change to `min` and `max` are to resolve an existing type match issue.

Reply:
Yeah although, the type isn't being used yet...